mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -241,6 +241,7 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 		FIELD f_jnxtuzdb hardness F
 		FIELD f_jvcpotqu isAir Z
 		FIELD f_lpessbtt pistonBehavior Lnet/minecraft/unmapped/C_mojvzhua;
+		FIELD f_lqzypovm spawnsBlockDustParticles Z
```

Comment:
```suggestion
		FIELD f_lqzypovm spawnsDustParticles Z
```
I think the `block` here is very redundant

Reply:
good point!